cluster-autoscaler/fix_gopath.sh

```diff
@@ -14,14 +14,11 @@ for item in $to_fix; do
    git commit -a -m from_staging
 done
 
-with_vendor=`find $GOPATH/src/ -type d | grep vendor | grep -v 'vendor/'`
-for item in $with_vendor; do
-    echo Removing vendor from $item
-    cd $item
-    cd ..
-    rm -rf $item
-    git commit -a -m no_vendor
-done
```

Comment:
By the way, with this change, only vendor/Godeps in cluster-autoscaler are removed ().

Reply:
The purpose of this loop is to remove vendor from dependencies that are going to be saved to Cluster Autoscaler's vendor (e.g kubernetes/kubernetes' vendor/)

Reply:
OK, let me revert this back.

Reply:
Wait, the original commands also removes vendors from vpa which is not expected. I think we should only removes vendors under CA

Reply:
It is intended to remove vendors from `$GOPATH/src/k8s.io/kubernetes` etc., not just Cluster Autoscaler.

If we want to exclude VPA (and addon resizer), we should explicitly check for those.

Reply:
> It is intended to remove vendors from $GOPATH/src/k8s.io/kubernetes etc., not just Cluster Autoscaler.

But actually not, it removes vendors for vpa. And it is not required to remove vendors from $GOPATH/src/k8s.io/kubernetes., godeps won't include vendored packages.

Reply:
Can we, for simplicity, keep it as is for a moment? We had mixed experience with godeps in the past. We would like to do the release today so right now we don't have enough capacity to validate that it will work without removals. 

We can get back to this discussion tomorrow :).

Reply:
>  And it is not required to remove vendors from $GOPATH/src/k8s.io/kubernetes., godeps won't include vendored packages.

I'll double-check this, but can you please revert it for now, so that we may proceed with the release? This hack works, and the cherry-pick with new version to 1.10 branch needs to be ready today. Let's continue this discussion after we made it in time for 1.10.3